data/transactions/logic/README_in.md

```diff
@@ -181,6 +187,13 @@ _available_.
  * Since v7, the account associated with any contract present in the
    `txn.ForeignApplications` field is _available_.
 
+ * A Box is _available_ if _any_ transaction in the same group
```

Comment:
nit: I think it would improve consistency and clarity if the CSP caveat was added here as well.

Reply:
Agree.